doc/topics/installation/windows.rst

```diff
@@ -414,16 +384,20 @@ Navigate to the root ``salt`` directory and install Salt.
 -------------------------------
 
 Navigate to the ``pkg\windows`` directory and run the ``build_pkg.bat``
-with the build version (2016.3) script.
+with the build version (2017.5.2) and the Python version as parameters.
```

Comment:
Oh, version here, too. There are a few more references with the `.5` version below. You get the idea. :)